ConvivaExampleApp/src/main/java/com/bitmovin/analytics/convivaanalyticsexample/MainActivity.java

```diff
@@ -87,18 +87,22 @@ protected void setupBitmovinPlayer() {
                 getApplicationContext(),
                 convivaConfig);
 
-
         MetadataOverrides metadata = new MetadataOverrides();
         metadata.setApplicationName("Bitmovin Android Conviva integration example app");
         metadata.setViewerId("awesomeViewerId");
-        Map<String, String> customInternTags = new HashMap<>();
-        customInternTags.put("contentType", "Episode");
-        metadata.setCustom(customInternTags);
+
+        Map<String, Object> standardTags = new HashMap<>();
```

Comment:
If the `setContentInfo` API accepts `Map<String, Object>` I would rather consider changing `customTags` as well to not take away a feature from the Conviva SDK 🤔

Reply:
This is the data type that setContentInfo accepts, so I didn't want to limit the customer: https://pulse.conviva.com/learning-center/content/sensor_developer_center/sensor_integration/android/android_stream_sensor.htm#:~:text=To%20update%20or%20amend%20pre%2Ddefined,metadata%20tags%20for%20video%20content.

For the custom tags, I left the type as is, because the `ContentMetadata` class (that I assume MetadataOverrides is based on) has it defined as Map<String, String>. We can change that if needed in the future IMO.

![image](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/assets/11734608/ec4c77e0-e2aa-4791-b4f8-9eac4db4a435)

Reply:
Ah, didn't realize Conviva's `reportPlaybackRequested` API takes `Map<String, Object>`. In this case I agree, we should change the handling of custom tags to also take an object.